README.md

```diff
@@ -186,10 +197,14 @@ If the `-r` option is used this list will also be pushed to GitHub, being the `-
     version
 ```
 
-### Update Build Config
+### Update Config For Release
 
-This task gathers information from the JIRA issues stored in the release file, updates the build configuration file and pushes 
-the changes to the gitlab repository to be merged.
+This task performs the following steps:
+* gathers information from the JIRA issues stored in the release file
+* updates the build configuration file
+    * pushes the changes to the gitlab repository to be merged
+* updates the CPaaS configuration script
```

Comment:
I think that we need more informations round that or that you add a link to the CPaaS project where this is described

Reply:
Included link to the cpaas documentation